packages/ingest/aws.js

```diff
@@ -63,8 +65,8 @@ function getExecutionArn(stateMachineArn, executionName) {
  */
 function getExecutionUrl(executionArn) {
   const region = process.env.AWS_DEFAULT_REGION || 'us-east-1';
-  return `https://console.aws.amazon.com/states/home?region=${region}` +
-         `#/executions/details/${executionArn}`;
+  return `https://console.aws.amazon.com/states/home?region=${region}`
+         + `#/executions/details/${executionArn}`;
```

Comment:
I think we should change the rule for operator-linebreaks, because strictly speaking this usage opens us up to automatic semicolon insertion bugs.

Reply:
Sounds reasonable.  I'll write a ticket to make that change.  I think that should be a separate change from this, though, since it's going to impact all of the packages, and this PR is only for the ingest package.

Reply:
I've created CUMULUS-973 to address that.